iroh-net/src/netcheck/reportgen/probes.rs

```diff
@@ -200,26 +193,22 @@ impl ProbePlan {
     pub(super) fn initial(derp_map: &DerpMap, if_state: &interfaces::State) -> Self {
         let mut plan = Self(BTreeSet::new());
 
-        let mut sorted_regions: Vec<_> = derp_map.regions().map(|r| (r.region_id, r)).collect();
-        sorted_regions.sort_by_key(|(id, _)| *id);
-
-        for (_, region) in sorted_regions {
-            let mut stun_ipv4_probes = ProbeSet::new(region.region_id, ProbeProto::StunIpv4);
-            let mut stun_ipv6_probes = ProbeSet::new(region.region_id, ProbeProto::StunIpv6);
+        for (_url, derp_node) in derp_map.nodes() {
+            let mut stun_ipv4_probes = ProbeSet::new(ProbeProto::StunIpv4);
+            let mut stun_ipv6_probes = ProbeSet::new(ProbeProto::StunIpv6);
 
             for attempt in 0..3 {
-                let derp_node = &region.nodes[attempt % region.nodes.len()];
                 let delay = DEFAULT_INITIAL_RETRANSMIT * attempt as u32;
 
-                if if_state.have_v4 && derp_node.ipv4.is_enabled() {
+                if if_state.have_v4 {
```

Comment:
I wonder how relevant this is?  I guess you could configure the Node to be only IPv4 or IPv6 in case you have some network trouble with one of them.  But maybe we just won't need this.

I don't think it would be a very hard thing to add tough.  We could add an `ipv4_enabled` and `ipv6_enabled` field to `DerpNode`, and allow getting that info from a query parameter in the URL.  Is this worth creating an issue for to keep track of and do as a follow-up?